test/framework/src/main/java/org/elasticsearch/bootstrap/BootstrapForTesting.java

```diff
@@ -88,40 +86,6 @@ public class BootstrapForTesting {
         IfConfig.logIfNecessary();
     }
 
-    static Map<String, URL> getCodebases() {
```

Comment:
I'm not super familiar but is the "init the priviledged operation" code above still relevant w/o security manager?

Reply:
I will remove that in a followup, it hits several tests.